sender should have the option to decide (#100)
intarchboard/draft-path-signals-collaboration: draft-iab-path-signals-collaboration.md

each entity over the collaboration or information sharing. This can
be achieved as discussed below.

> The sender should have an option to decide that it wants to send the information
> to a specific entities or set of entities.

Reviewer: ```suggestion
The sender needs to decide that it is willing to send information to a specific entity or
set of entities.
```

Any passing of information or request for an action needs to be explicit,
and use signalling mechanisms that are designed for the purpose.
Merely sending a particular kind of packet to a destination should not
be interpreted as an implicit agreement.

At the same time, the recipient of information or the target of a

> request should have the option to agree or deny to receiving the information. It

Reviewer: ```suggestion
request needs to decide that it is willing to receive the information. It
```
